website_lazy_load_image/__manifest__.py

```diff
@@ -8,7 +8,7 @@
     'license': 'AGPL-3',
     'website': 'https://github.com/oca/website',
     'category': 'Website',
-    'version': '12.0.1.0.0',
+    'version': '12.0.1.0.1',
```

Comment:
this is not required as it will be done via bot. Also, ATM there's release on pypi w/ `nobump` option.